mesheryctl/internal/cli/root/root.go

```diff
@@ -61,10 +60,19 @@ mesheryctl -v [or] --verbose
 	`,
 	// Uncomment the following line if your bare application
 	// has an action associated with it:
-	PreRunE: func(cmd *cobra.Command, args []string) error {
-		log.Println("Args passed in", args)
+	DisableSuggestions: true,
+	Args: func(cmd *cobra.Command, args []string) error {
+
+		if len(args) == 0 {
+			return cmd.Help()
+		}
+		return nil
```

Comment:
This should be moved to the `RunE` block, right now I'm getting the following error.

```
➜ go run mesheryctl/cmd/mesheryctl/main.go    
Meshery is the service mesh management plane, providing lifecycle, performance, and configuration management of service meshes and their workloads.

Usage:
  mesheryctl [flags]
  mesheryctl [command]

Examples:

// Base command
mesheryctl

// Display help about command/subcommand
mesheryctl --help
mesheryctl system start --help

// For viewing verbose output
mesheryctl -v [or] --verbose
	

Available Commands:
  app         Service Mesh Apps Management
  completion  Output shell completion code
  exp         Experimental commands for mesheryctl
  help        Help about any command
  mesh        Service Mesh Lifecycle Management
  pattern     Service Mesh Patterns Management
  perf        Performance Management
  system      Meshery Lifecycle Management
  version     Version of mesheryctl

Flags:
      --config string   path to config file (default "/Users/nishantwrp/.meshery/config.yaml")
  -h, --help            help for mesheryctl
  -v, --verbose         verbose output

Use "mesheryctl [command] --help" for more information about a command.
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/layer5io/meshery/mesheryctl/internal/cli/root.glob..func3(0x108cd6640?, {0x108ecfb90?, 0x0?, 0x0?})
	/Users/nishantwrp/Dev/open-source/layer5/meshery/mesheryctl/internal/cli/root/root.go:74 +0x188
github.com/spf13/cobra.(*Command).execute(0x108cd6640, {0x140001921c0, 0x0, 0x0})
	/Users/nishantwrp/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:872 +0x4d0
github.com/spf13/cobra.(*Command).ExecuteC(0x108cd6640)
	/Users/nishantwrp/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990 +0x354
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/nishantwrp/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
github.com/layer5io/meshery/mesheryctl/internal/cli/root.Execute()
	/Users/nishantwrp/Dev/open-source/layer5/meshery/mesheryctl/internal/cli/root/root.go:89 +0x5c
main.main()
	/Users/nishantwrp/Dev/open-source/layer5/meshery/mesheryctl/cmd/mesheryctl/main.go:24 +0x20
exit status 2
```